### ISSUE TITLE
Run as other users?

### ISSUE THREAD

**Issue by zenithyr** [cronicle-edge/cronicle-edge#128]
I didn't get the point of using dockerized Cronicle to run jobs.
I used to install Cronicle directly on bare metals as the root user. Then I create multiple users in Cronicle for the root user to switch into.
But is this possible to do so in docker? If not, is the docker version only suitable to run independent tasks like CI/CD?

**Comment by mikeTWC1984** [comment]
Yes, it work the same as bare metal. You can run cronicle engine as root then run plugins under non-privileged user. It's actually how I run it

**Comment by zenithyr** [comment]
> Yes, it work the same as bare metal. You can run cronicle engine as root then run plugins under non-privileged user. It's actually how I run it

It did work. Many thanks. 
Seems I have to docker exec into the container first, and create the corresponding uids manually on `/etc/passwd` file. Otherwise plugins report a user not found error.
Do you have a better approach?

**Comment by mikeTWC1984** [comment]
No need to touch etc/password, you can create passwordless user in docker file

https://github.com/cronicle-edge/cronicle-edge/blob/main/Dockerfile

**Comment by helmut72** [comment]
I guess he mean the `/etc/passwd`file to create users at all. `/etc/passwd` doesn't store passwords, it's `/etc/shadow`.
